examples/pod_exec_terminal_size.rs

```diff
@@ -0,0 +1,134 @@
+use futures::{channel::mpsc::Sender, SinkExt, StreamExt};
```

Comment:
we should rename this to `pod_shell_crossterm` to match the other example and highlight functionality